src/libraries/EndpointStructs.sol

```diff
@@ -7,6 +7,9 @@ library EndpointStructs {
     using BytesParsing for bytes;
 
     error PayloadTooLong(uint256 size);
+    error IncorrectPrefix(bytes4 prefix);
+
+    bytes4 constant NTT_PREFIX = 0x994E5454;
```

Comment:
Can you add a comment to explain the prefix. I assume it's the first 4 bytes of the hash of some relevant string?

Reply:
Ah my comment got accidentally removed. It's meant to be `0x99'N''T''T'`